scripts/lib/libvirt/templates/cpu-default.xml

```diff
@@ -0,0 +1,16 @@
+<cpu match='minimum'>
+    <model>qemu64</model>
+    <feature policy='require' name='fxsr_opt'/>
+    <feature policy='require' name='mmxext'/>
+    <feature policy='require' name='lahf_lm'/>
+    <feature policy='require' name='sse4a'/>
+    <feature policy='require' name='abm'/>
+    <feature policy='require' name='cr8legacy'/>
+    <feature policy='require' name='misalignsse'/>
+    <feature policy='require' name='popcnt'/>
+    <feature policy='require' name='pdpe1gb'/>
+    <feature policy='require' name='cx16'/>
+    <feature policy='require' name='3dnowprefetch'/>
+    <feature policy='require' name='cmp_legacy'/>
+    <feature policy='require' name='monitor'/>
+  </cpu>
```

Comment:
Are you sure?!  Seems very unlikely the code would automatically eat whitespace, but if it does then that is a bug which needs to be understood not ignored ...